mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule SmartCity.MixProject do
   def project do
     [
       app: :smart_city,
-      version: "3.14.1",
+      version: "3.15.0",
```

Comment:
Also, we may want to update the version on the `SmartCity.Data` struct itself (from 0.5 to 0.6?) Not sure we have normed on when to do that.  Just wanted to make sure we're thinking about it.